guide/daily-spending-wallet/backup-and-recovery/recovery.md

```diff
@@ -33,47 +33,395 @@ Illustration sources:
 
 # Wallet recovery
 
+There are several reasons why a user might need to restore their bitcoin wallet. They may have lost their device or accidentally deleted their wallet application. Perhaps they have simply bought a new phone and wish to migrate their wallet over. Regardless of why, it’s important that your wallet application has an easy way for users to recover their funds.
```

Comment:
```suggestion
There are several reasons why a user might need to restore their bitcoin wallet. They may have lost their phone or accidentally deleted their wallet application. Perhaps they have simply bought a new phone and wish to migrate their wallet over. Regardless of why, it’s important that your wallet application has an easy way for users to recover their funds.
```

Use phone instead of device so its more in-line with the scope of this reference design